app/views/admin_mailer/feedback.html.erb

```diff
@@ -1,15 +1,27 @@
 <% content_for :message do %>
 
-  <p><%= style_bold("Here's some user feedback on the archive!") %></p>
-  
+  <p><%= style_bold("Here's some user feedback on the Archive!") %></p>
+  <% if @username.blank? %>
+    <p>Username: Anonymous user</p>
+    <% else %>
+    <p>Username: <%= raw @username %></p>
+  <% end %>
+  <% unless @email.blank? %>
+    <p>Email address: <%= raw @email %></p>
+  <% end %>
+  <p>
+    Language: <%= @language %>
```

Comment:
Since I'm already making you change things, can you put the p tags on the same line as language here, like the other parts? `<p>Language: <%= @language %></p>`